web/client/components/data/featuregrid/DockedFeatureGrid.jsx

```diff
@@ -295,6 +297,21 @@ const DockedFeatureGrid = React.createClass({
     selectFeatures(features) {
         this.props.selectAllToggle();
         this.props.selectFeatures(features);
+    },
+    backToSearch() {
+        this.props.onToggleDrawer();
+        this.props.onClose();
+    },
+    limitDockHeight(size) {
+        const minSize = 0.25;
```

Comment:
Should they be also action and reducer or only props of the class? (minSize and maxSize)